doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1154 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f5c2fbd-31d7-4262-94c8-1f6cea6f5938",
+   "metadata": {},
+   "source": [
+    "# Running N Independent Trials in Parallel with Ray AIR Tune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdb02a92-a603-48b8-a23e-887ecaaf5470",
+   "metadata": {},
+   "source": [
+    "# Introduction \n",
```

Comment:
Make sure you talk about WHAT you're doing (from the data scientist perspective).

Right now, the text covers more of "HOW" you're doing it.